avalon/maya/compat.py

```diff
@@ -0,0 +1,29 @@
+"""Compatibility
+
+This module is
```

Comment:
_This module is_ what? :D

Maybe you just stopped typing there for a second. Hehe.